tests/components/wled/test_sensor.py

```diff
@@ -81,26 +120,70 @@
     assert entry
     assert entry.unique_id == "aabbccddeeff_free_heap"
 
+    state = hass.states.get("sensor.wled_rgb_light_wifi_signal")
+    assert state
+    assert state.attributes.get(ATTR_ICON) == "mdi:wifi"
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == UNIT_PERCENTAGE
+    assert state.state == "76"
 
+    entry = registry.async_get("sensor.wled_rgb_light_wifi_signal")
+    assert entry
+    assert entry.unique_id == "aabbccddeeff_wifi_signal"
+
+    state = hass.states.get("sensor.wled_rgb_light_wifi_rssi")
+    assert state
+    assert state.attributes.get(ATTR_ICON) == "mdi:wifi"
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == SIGNAL_DBM
+    assert state.state == "-62"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5221899)